### PR TITLE
Remove unneeded `Friend::file_id` field

### PR DESCRIPTION
**Pull request by DarkSide666**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ If you have a Model such as `Friend` and you wish to upload friend's photo, File
 
 ```php
 // in Friend::init();
-$this->addField('file_id', [\Atk4\Filestore\Field\FileField::class]);
+$this->addField('photo', [\Atk4\Filestore\Field\FileField::class]);
```
Comment (mvorisek) [inline README.md]:
```suggestion
$this->addField('photo_file_id', [\Atk4\Filestore\Field\FileField::class]);
```

Comment (DarkSide666) [reply README.md]: It's not `id`, it's file `token`. Filestore do not link with other models trough `id` field, but using `token` field instead.
Why that was done like that I'm not exactly sure, but probably to completely decouple filestore tables from other tables in DB.

Comment (DarkSide666) [reply README.md]: But can be `photo_file_id` too here. This is mostly deprecated documentation anyway.

Comment (mvorisek) [reply README.md]: then it should be `photo_file_token` ;-)

Comment (DarkSide666) [reply README.md]: ok :)

Comment (mvorisek) [reply README.md]: and on line 109?

Comment (DarkSide666) [reply README.md]: done :)

**Pull request merged**